content/blog/2021/02/camel-quarkus-release-1.7.0/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Camel Quarkus 1.7.0 Released"
+date: 2021-02-24
+authors: ["ppalaga"]
+categories: ["Releases", "Camel Quarkus"]
+preview: "Camel Quarkus 1.7.0 Released"
+summary: "The highlights of Camel Quarkus 1.7.0"
+---
+
+<sub><sup>Original image by <a href="https://commons.wikimedia.org/wiki/User:99of9">Toby Hudson</a> <a href="https://creativecommons.org/licenses/by-sa/3.0">CC BY-SA 3.0</a> via <a href="https://en.wikipedia.org/wiki/Camel_racing#/media/File:CamelRacingCamelCup2009Heat.JPG">Wikipedia</a></sup></sub>
+
+We are pleased to announce the release 1.7.0 of Camel Quarkus! Here are the highlights.
+
+## Camel 3.8.0 and Quarkus 1.12.0.Final
+
+We stand on the shoulders of two giants. Please check the release announcements for
+[Camel 3.8.0](/blog/2021/02/Camel38-Whatsnew/) and [Quarkus 1.12.0.Final](https://quarkus.io/blog/quarkus-1-12-0-final-released/) to learn about the news they bring.
+
+The [change of default packaging format](https://quarkus.io/blog/quarkus-1-12-0-final-released/#fast-jar-as-default)
+to Fast JAR in Quarkus 1.12 is worth mentioning also here. Before Camel Quarkus 1.7.0 you used to start your application
+in JVM mode via something like
+
+```sh
+$ java -jar target/*-runner.jar
+```
+
+and now since Camel Quarkus 1.7.0, you have to use
+
+```sh
+$ java -jar target/quarkus-app/quarkus-run.jar
+```
+
+Your Docker files and deployment scripts might need an adjustment when you upgrade to Camel Quarkus 1.7.0.
+
+You can go back to the original behavior by setting `quarkus.package.type=legacy-jar` in your
+`application.properties`.
+
+## Support for more Camel components
+
+Camel Quarkus brings support for six new Camel components:
+
+* [Azure Event Hubs](/camel-quarkus/latest/reference/extensions/azure-eventhubs.html)
+* [Kamelet](/camel-quarkus/latest/reference/extensions/kamelet.html)
+* [OAI-PMH](/camel-quarkus/latest/reference/extensions/oaipmh.html)
+* [Spring RabbitMQ](/camel-quarkus/latest/reference/extensions/spring-rabbitmq.html)
+* [XML Security Sign and Verify](/camel-quarkus/latest/reference/extensions/xmlsecurity.html)
+* [JFR](/camel-quarkus/latest/reference/extensions/jfr.html) (JVM only)
+
+Components newly supported in native mode::
```

Comment:
A small detail here, the bulleted list does not appear well in the preview. Maybe due to the double ":"

Reply:
Thanks, fixed in 72fd09d